recipe/meta.yaml

```diff
@@ -1,41 +1,42 @@
-{% set version = "1.6.3" %}
+{% set name = "g2clib" %}
+{% set version = "1.7.0" %}
 
 package:
-  name: g2clib
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: http://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/g2clib-{{ version }}.tar
-  sha256: 83e4f2061b3d4a8bd431ba860dda8c5cf103b25f42d265b7b724b9acafad177c
+  url: https://github.com/NOAA-EMC/NCEPLIBS-g2c/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 73afba9da382fed73ed8692d77fa037bb313280879cd4012a5e5697dccf55175
 
 build:
-  number: 1
-  skip: True  # [win]
+  number: 0
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - make
+    - cmake
+    - pkg-config >=0.21
+    - make                  # [not win]
+    - ninja                 # [win]
   host:
     - libpng
     - zlib
-    - jasper
+    - jasper >=2.0.25
   run:
     - libpng
     - zlib
-    - jasper
-
-test:
-  commands:
-    - test -f ${PREFIX}/lib/libgrib2c.a  # [unix]
```

Comment:
@timcera, I think we still want some kind of test.  If this should be for a dynamic library now, you can use a check similar to:
https://github.com/conda-forge/netcdf-fortran-feedstock/blob/main/recipe/meta.yaml#L86
```

test:
  commands:
    - test -f ${PREFIX}/lib/libgrib2c${SHLIB_EXT}  # [not win]
```

You could also make sure static libraries are *not* present with:
```
   - test ! -f ${PREFIX}/lib/libgrib2c.a  # [not win]
```